demo/src/main/java/com/novoda/landingstrip/CustomLandingStrip.java

```diff
@@ -2,17 +2,17 @@
 
 import android.os.Bundle;
 import android.support.v4.view.ViewPager;
-import android.support.v7.app.ActionBarActivity;
-
+import android.support.v7.app.AppCompatActivity;
 import com.novoda.landing_strip.R;
 import com.novoda.landingstrip.setup.fragment.DemoFragmentPagerAdapter;
 
-public class CustomLandingStrip extends ActionBarActivity {
+public class CustomLandingStrip extends AppCompatActivity {
```

Comment:
These changes are unrelated to the purpose of the PR. Small focused PRs ftw :smile_cat: 

(not a blocker for me in this _particular_ case)